docker-images/artifacts/kafka-thirdparty-libs/3.1.x/pom.xml

```diff
@@ -23,7 +23,7 @@
         <kafka-mirror-maker-2-extensions.version>1.2.0</kafka-mirror-maker-2-extensions.version>
         <kafka-kubernetes-config-provider.version>1.0.1</kafka-kubernetes-config-provider.version>
         <kafka-env-var-config-provider.version>1.0.0</kafka-env-var-config-provider.version>
-        <jmx-prometheus-javaagent.version>0.17.0</jmx-prometheus-javaagent.version>
+        <jmx-prometheus-javaagent.version>0.17.2s</jmx-prometheus-javaagent.version>
```

Comment:
I guess `0.17.2s` is a typo and it should be `0.17.2`?

Reply:
Yes.  Fixed